bindings/nodejs/examples/how_tos/evm/send_nft_to_evm.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2021-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    Client,
+    initLogger,
+    Utils,
+    UnlockCondition,
+    Ed25519Address,
+    SenderFeature,
+    MetadataFeature,
+    Wallet,
+    AddressUnlockCondition,
+    AliasAddress,
+    NftOutput,
+} from '@iota/sdk';
+import {
+    prepareMetadata
+} from './utils';
+
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./how_tos/evm/send_nft_to_evm.ts
+
+const amount = 1000000;
+const gas = 10000;
+const toEVMAddress = '0x48e28C1681BBb92a2E5874113bc740cC11A0FD7a';
+const chainAddress = 'rms1pr75wa5xuepg2hew44vnr28wz5h6n6x99zptk2g68sp2wuu2karywgrztx3';
+
+// Build ouputs with all unlock conditions
+async function run() {
+    initLogger();
+
+    const client = new Client({});
```

Comment:
New Client without node will use default options which might be wrong depending on which network is used
Remove this and just get the client from the wallet instead when needed
`await wallet.getClient()`